resources/views/entities/components/menu.blade.php

```diff
@@ -2,6 +2,15 @@
 @php $modelMenuItems = $model->menuItems(); @endphp
 <div class="hidden-xs">
 @foreach ($modelMenuItems as $section => $menuItems)
+    @if($section == 2)
+        @php
+            $items = array_column($menuItems, 'name');
+            foreach ($items as $key => $item) {
+                $items[$key] = __($item);
+            }
+            array_multisort($items, SORT_ASC, $menuItems);
```

Comment:
SORT_ASC doesn't work with accented characters (Russian, French). 

1. Switch to French
2. open `lang/fr/crud.php` and find `tabs.inventory` and replace it with `Înventaire` (accented uppercase letter)
3. Refresh the browser, see `Înventaire` all at the end, when it should be in second position.

See how it's done in the quick creator's entity type list to sort alphabetically for international languages :)